DependencyInjection/CmfRoutingExtension.php

```diff
@@ -240,13 +240,15 @@ public function loadPhpcrProvider($config, XmlFileLoader $loader, ContainerBuild
 
         $container->setParameter(
             $this->getAlias() . '.dynamic.persistence.phpcr.route_basepaths',
-            $config['route_basepaths']
+            array_values(array_unique($config['route_basepaths']))
```

Comment:
would this not be the job of the test? but well, should not hurt mutch to elminiate holes in the keys, could lead to confusion somewhere.